egs/fisher_callhome_spanish/asr1/run.sh

```diff
@@ -0,0 +1,367 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=0        # start from 0 if you need to start from data preparation
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+seed=1
+
+# feature configuration
+do_delta=false
+
+# network architecture
+# encoder related
+etype=vggblstmp     # encoder architecture type
+elayers=3
+eunits=1024
+eprojs=1024
+subsample=1_1_1 # skip every n frame from input to nth layers
+# decoder related
+dlayers=1
+dunits=1024
+# attention related
+atype=location
+adim=320
+awin=5
+aheads=4
+aconv_chans=10
+aconv_filts=100
+
+# hybrid CTC/attention
+mtlalpha=0.5
+
+# label smoothing
+lsm_type=
+lsm_weight=0.05
+
+# minibatch related
+batchsize=24
+maxlen_in=800  # if input length  > maxlen_in, batchsize is automatically reduced
+maxlen_out=150 # if output length > maxlen_out, batchsize is automatically reduced
+
+# optimization related
+opt=adadelta
+epochs=15
+
+# rnnlm related
+use_wordlm=true     # false means to train/use a character LM
+lm_vocabsize=65000  # effective only for word LMs
```

Comment:
If you only use the transcriptions, 65k would be a bit too large, but you can test it later.

Reply:
Yes. I agree with you as we also talked during the meeting. Let me test this some time.
I would like to make sure this (i.e. setting larger vocab size than vocabularies seen in the training data) doesn't affect the decoding result. Do you know about this @takaaki-hori ?